ThingIFSDK/ThingIFSDK/StatePredicate.swift

```diff
@@ -48,10 +48,13 @@ open class StatePredicate: Predicate {
      - Returns: a NSDictionary instance
      */
     open func makeDictionary() -> [ String : Any ]  {
```

Comment:
@yongpingchen It will become internal in other PR. I will fix `StatePredicate` in next PR:+1: